teuthology/orchestra/remote.py

```diff
@@ -191,10 +196,7 @@ def run(self, **kwargs):
 
         TODO refactor to move run.run here?
         """
-        if not self.ssh or \
-           not self.ssh.get_transport() or \
-           not self.ssh.get_transport().is_active():
-            self.reconnect()
```

Comment:
@tchaikov why have you remove `self.reconnect()` it was doing very useful thing when rebooting a node

Reply:
the `ensure_online()` does not do this.

Reply:
> it was doing very useful thing when rebooting a node

could you define "very useful thing"?

Reply:
the name of function "reconnect" is self explanatory, I suppose

Reply:
it does. "reconnect" connect without passing create_key or host_key.